python/lsst/obs/base/formatters/fitsExposure.py

```diff
@@ -212,6 +215,16 @@ def readComponent(self, component, parameters=None):
 
                 if component == "dimensions" and thisComponent is not None:
                     thisComponent = thisComponent.getDimensions()
+
+                if component == "FilterLabel":
```

Comment:
The component is `filterLabel` (lower case). Should you do a similar check for the `filter` component, or is this already covered because it's derived from `filterLabel`?

Reply:
> The component is `filterLabel` (lower case).

:+1: And hmm, now you've got me wondering whether my tests are doing what I think they're doing.

> Should you do a similar check for the filter component, or is this already covered because it's derived from filterLabel?

I was not planning to check `Filter` at all, because I didn't think I could predict what the correct content would be from a `Filter` given the data ID (especially if "correct" might mean "what it was before filter standardization").

Reply:
Ah, I see what happened with `FilterLabel` vs `filterLabel` - this is a relic from an earlier version, and the (now) operative check does use `filterLabel` elsewhere.  I'll remove this `if` block entirely.